client/src/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import '../index.css';
+import { accountRepository } from '@/_services';
+import { authenticationRepository } from '@/_services';
+import config from 'config';
+const signalR = require('@aspnet/signalr')
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            accountId: null,
+            currentPlan: null,
+            numberOfUsers: null,
+            maximumNumberOfUsersAllowed: null,
+            pricePerMonth: null,
+            hub: null
+        };
+
+        this.logout = this.logout.bind(this);
+        this.upgradePlan = this.upgradePlan.bind(this);
+    }
+
+    componentDidMount() {
+        const hub = this.configureUserHub();
+        this.start(hub);
+
+        this.createAndGetAccount();
+    }
+
+    createAndGetAccount() {
+        accountRepository.create()
```

Comment:
Every time we go to the Dashboard, we'll make an account and retrieve it for the user since there's no way in the UI to make an account. In the future I'd probably expose a button in the project to make an account first.